config/initializers/europeana.rb

```diff
@@ -17,3 +17,5 @@
   ENV['EUROPEANA_MEDIA_PROXY'] = ENV['EDM_IS_SHOWN_BY_PROXY']
 end
 Rails.application.config.x.europeana_media_proxy = ENV['EUROPEANA_MEDIA_PROXY'] || 'http://proxy.europeana.eu'
+
+Rails.application.config.x.europeana_1914_1918_url = ENV['EUROPEANA_1914_1918_URL'] || 'http://acceptance.1418.eanadev.org'
```

Comment:
This should default to the new hostname assigned to the production deployment of 1914-1918, but that has not yet been decided. (See https://europeanadev.assembla.com/spaces/europeana-npc/tickets/2151.)